### PR TITLE
Works around the pool/vs stats failure on 15.x Note: this does NOT fix the issue.

### PR DESCRIPTION
**Pull request by timriker**

### CODE REVIEW

File: bigipreport.ps1 [hunk 1]
```diff
@@ -1537,14 +1547,15 @@ function Get-LTMInformation {
         $ObjTempVirtualServer.availability = $VirtualServerStates[$i].availability_status
         $ObjTempVirtualServer.enabled = $VirtualServerStates[$i].enabled_status
 
+        Try {
+	# FIXME: 15.x fails to load vittual server statistics
```
Comment (epacke) [inline bigipreport.ps1:1551]: I assume vittual should be virtual?

**Review by epacke — APPROVED** [review]
Just a spelling error in a comment. Approved!

File: bigipreport.ps1 (hunk 1 above)
Comment (timriker) [reply bigipreport.ps1:1551]: Are you fixing the spelling or should I?

Comment (timriker) [reply bigipreport.ps1]: I got it. :)

**Pull request merged**